Helper/FeatureSwitch/Definitions.php

```diff
@@ -456,5 +461,11 @@ class Definitions
             self::DEFAULT_VAL_KEY => false,
             self::ROLLOUT_KEY     => 100
         ],
+        self::M2_DISALLOW_SPECIFIC_ORDER_STATUS_UPDATE => [
+            self::NAME_KEY        => self::M2_DISALLOW_SPECIFIC_ORDER_STATUS_UPDATE,
+            self::VAL_KEY         => true,
+            self::DEFAULT_VAL_KEY => true,
```

Comment:
DEFAULT_VAL_KEY should be false

Reply:
@vitaliyreznikov By default we do not allow updating order status from pending to new again. So this value is set to true.

Reply:
Please check how other feature switches enable by default look like. It should be 
```
            self::VAL_KEY         => true,
            self::DEFAULT_VAL_KEY => false,
            self::ROLLOUT_KEY     => 100
```